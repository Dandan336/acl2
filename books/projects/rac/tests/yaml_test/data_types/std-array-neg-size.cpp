#include <array>
using namespace std;

// RAC begin

typedef array<int, -4> invalid_array;

int foo()
{
  return 0;
}

// RAC end
