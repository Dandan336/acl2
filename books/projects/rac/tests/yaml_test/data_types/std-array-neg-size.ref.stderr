std-array-neg-size.cpp:6: Non-constant array dimension
